src/Behat/Context.php

```diff
@@ -629,35 +583,33 @@ private function assertSelectedValue(NodeElement $element, string $value, string
      */
     private function assertInputValue(NodeElement $element, string $value, string $selector = 'input'): void
     {
-        $input = $element->find('css', $selector);
-        if (!$input) {
-            throw new Exception('Input not found in selector: ' . $selector);
-        }
-        $inputValue = $input->getValue();
-        if ($inputValue !== $value) {
+        if ($this->assertElementInElement($element, $selector)->getValue() !== $value) {
             throw new Exception('Input value not is not: ' . $value);
         }
     }
 
     private function getScopeBuilderRuleElem(string $ruleName): NodeElement
     {
-        $rule = $this->getSession()->getPage()->find('css', '.vqb-rule[data-name=' . $ruleName . ']');
-        if (!$rule) {
-            throw new Exception('Rule not found: ' . $ruleName);
+        return $this->assertElementInPage('.vqb-rule[data-name=' . $ruleName . ']');
+    }
+
+    private function assertElementInPage(string $selector, string $method = 'css'): NodeElement
+    {
+        $element = $this->getSession()->getPage()->find($method, $selector);
+        if ($element === null) {
+            throw new Exception('No element found with class name: ' . $selector);
```

Comment:
here you call it "class name", in exception below "selector", if it can be more than class name, use "selector" keyword instead

Reply:
Done!